core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -254,12 +287,15 @@ private[spark] class SparkFileAlreadyExistsException(
 private[spark] class SparkFileNotFoundException(
     errorClass: String,
     errorSubClass: Option[String] = None,
-    messageParameters: Array[String])
+    messageParameters: Map[String, String])
```

Comment:
It is tested by QueryExecutionErrorsSuite "RENAME_SRC_PATH_NOT_FOUND: rename the file which source path does not exist"